Makefile

```diff
@@ -17,8 +18,10 @@ build:
 	cargo build --release
 
 # Using --release for testing because wasm can otherwise be slow.
-test: $(BUILTIN_ACTORS_BUNDLE)
-	FM_BUILTIN_ACTORS_BUNDLE=$(BUILTIN_ACTORS_BUNDLE) cargo test --release --workspace --exclude smoke-test
+test: $(BUILTIN_ACTORS_BUNDLE) $(IPC_ACTORS_ABI)
```

Comment:
It may be worth checking the `out` in with the IPC actors repo if building on CI takes too long.